src/components/HighTable/HighTable.module.css

```diff
@@ -336,22 +361,6 @@
     filter: blur(4px);
   }
 
-  /* pending table state */
-  thead th::before {
-    content: "";
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 4px;
-    background-color: #706fb1;
-    z-index: var(--header-progress-z-index);
-  }
-  /* TODO(SL): add a global pending state? */
-  /* .pending thead th::before {
-    animation: shimmer 2s infinite linear;
```

Comment:
looks like shimmer animation is no longer referenced by anything

Reply:
yes, for a long time, it's not related to this PR. But I can look at it to add it again if needed

Reply:
see #154